### PR TITLE
fix(list-view): Fixed off-by-one bug in ListView

### PR DESCRIPTION
**Pull request by macrotim**
I made a few fixes before March 8's sprint demo including...
* Fixed a bug where ListView rendered N-1 items. The first item was always omitted.
* Updated SkeletonListView (the ghost state) to render more rows. 
* Revised the ListView README to make it easier to spot bugs e.g. the stubbed data more clearly distinguishes header from rows.

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @macrotim, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

File: src/features/list-view/examples/ListViewExamples.js [hunk 1]
```diff
@@ -7,21 +7,21 @@ import ListView from '../ListView';
...
-const getGridHeader = columnIndex => generateRandomString(columnIndex);
+const getGridHeader = columnIndex => `Header ${generateRandomString(columnIndex)}`;
 
-const getGridCell = ({ columnIndex, rowIndex }) => generateRandomString(columnIndex + rowIndex);
+const getGridCell = ({ columnIndex, rowIndex }) => `Row ${rowIndex}, Column ${columnIndex}`;
 
 const ListViewExamples = () => {
     return (
-        <div style={{ height: '1000px' }}>
+        <div style={{ height: '300px' }}>
```
Comment (ChenCodes) [inline src/features/list-view/examples/ListViewExamples.js]: Could this height be moved to a constant?

File: src/features/list-view/examples/ListViewExamples.js [hunk 2]
```diff
@@ -7,21 +7,21 @@ import ListView from '../ListView';
...
             <AutoSizer>
                 {({ height, width }) => (
                     <ListView
                         columnCount={1000}
                         height={height}
                         getGridCell={getGridCell}
                         getGridHeader={getGridHeader}
-                        rowCount={35}
+                        rowCount={1000}
```
Comment (ChenCodes) [inline src/features/list-view/examples/ListViewExamples.js]: Could this and columnCount also be moved to be a constant?

**Comment by priyajeet** [comment]
"I made a few fixes before March 8's sprint demo." is not a useful description

File: src/features/list-view/examples/ListViewExamples.js (hunk 1 above)
Comment (macrotim) [reply src/features/list-view/examples/ListViewExamples.js]: I'm less concerned given that it's a README.

Do you see any functional or readability issues with the change-set?

File: src/features/list-view/examples/ListViewExamples.js (hunk 2 above)
Comment (macrotim) [reply src/features/list-view/examples/ListViewExamples.js]: Same.

**Comment by macrotim** [comment]
@priyajeet Good point. I updated the description with more info.

**Pull request merged**